workshops/read_the_internet/README.md

```diff
@@ -0,0 +1,175 @@
+---
+name: 'Reading the Internet'
+description: 'Use Python to read HTML and extract any information you can find!'
+author: '@RafaelCenzano'
+img: 'https://cloud-p5pdve0kf.vercel.app/2webscrape.jpg'
+---
+
+We can read all the content on websites, but wouldn't it be cool if we could automatically read specific data from websites whenever you wanted? If this interested you, this workshop is for you. I will be showing you how to use webscraping in Python.
+
+*Note: not all webpages can be read using this technique. If data is loaded in using ajax or after the inital page request this method may not work.*
+
+[Final Result and Code](https://repl.it/@SavageCoder77/Read-The-Internet#main.py)
+
```

Comment:
Maybe also include a screenshot of the covid case numbers displayed in the terminal